src/upgradeOptions.js

```diff
@@ -0,0 +1,18 @@
+const define = defaults => opts => Object.assign(opts, defaults);
+
+const updaters = {
+  __proto__: null,
```

Comment:
curious - what's the purpose of setting `__proto__` to `null` here? just to handle the case that `updaters[result[0]]` could be called with `result[0]` as something in the prototype?

Reply:
like “toString”